qiita_db/sql_connection.py

```diff
@@ -1047,10 +1065,44 @@ def rollback(self):
         except Exception:
             self._connection.close()
             raise
+        # Execute the post rollback commands
+        self._cmds_executor(self._post_rollback_funcs, "rollback")
 
     @property
     def index(self):
         return len(self._queries) + len(self._results)
 
+    @_checker
+    def add_post_commit_func(self, func):
```

Comment:
Since this is some really nifty functionality, I would probably describe this feature as part of the docstring ...